src/Aspire.Hosting.AWS/README.md

```diff
@@ -157,6 +157,107 @@ builder.AddProject<Projects.Frontend>("Frontend")
        .WithReference(construct, c => c.Url, "Url");
 ```
 
+## Integrating AWS Lambda Local Development
+
+You can develop and test AWS Lambda functions locally within your .NET Aspire application. This enables testing Lambda functions alongside other application resources during development.
+
+![\[\]!(.)](Resources/lambda.png)
+
+
+### Adding Lambda Functions
+
+To add a Lambda function to your .NET Aspire AppHost, use the `AddAWSLambdaFunction` method. The method supports both executable Lambda functions and class library Lambda functions:
+
+```csharp
+#pragma warning disable CA2252 // This API requires opting into preview features
+
+// Add an executable Lambda function
+builder.AddAWSLambdaFunction<Projects.ExecutableLambdaFunction>(
+    "MyLambdaFunction", 
+    handler: "ExecutableLambdaFunction")
+    .WithReference(awsConfig);
```

Comment:
Can you include the full sample that includes the creation of `awsConfig`. Don't want users to be confused what that object is.

Reply:
updated